cfgov/jinja2/v1/_includes/organisms/filterable-list.html

```diff
@@ -0,0 +1,184 @@
+
+        {# ADD THE NOTIFICATION, IF NEEDED. #}
+
+        {% set notification_opts = {
+            'type': 'default',
+            'is_visible': false,
+            'message': '',
+            'explanation': ''
+        } %}
+
+        {% for field in form %}
+            {% if field.errors %}
+                {% for error in field.errors %}
+                    {% if notification_opts.update({
+                        'type': 'error',
+                        'is_visible': true,
+                        'message': notification_opts.message + field.label + ': ' + error + '<br>' | safe
+                    }) %}{%endif%}
+                {% endfor %}
+            {% endif %}
+        {% endfor %}
+        {% if 'filterable-list' in value.form_type and posts is defined %}
+            {% if form.is_valid() %}
+                {% set count = posts.paginator.count %}
+                {% set notification_is_visible = true %}
+                {% if count == 0 %}
+                    {% if notification_opts.update({
+                        'type': 'warning',
+                        'is_visible': true,
+                        'message': 'Sorry, there were no results based on your filter selections.',
+                        'explanation': 'Please reset the filter or change your selections and try again.'
+                    }) %}{%endif%}
+                {% else %}
+                    {% if notification_opts.update({
+                        'type': 'success',
+                        'is_visible': true,
+                        'message': count ~ ' filtered result' ~ count|pluralize('s')
+                    }) %}{%endif%}
+                {% endif %}
+            {% endif %}
+        {% endif %}
+
+        <div class="o-filterable-list_notification
+                    block
+                    block__padded
+                    block__sub
+                    {% if not has_active_filters %}
+                    u-hidden
+                    {% endif %}">
+            {{ notification.render(
+                notification_opts.type,
+                notification_opts.is_visible,
+                notification_opts.message,
+                notification_opts.explanation
+            ) }}
+        </div>
+
+        {# ADD THE FILTER RESULTS, IF THERE ARE ANY. #}
+
+        <section class="o-filterable-list_results
+                        block
+                        block__sub
+                        {% if has_active_filters %}
+                        block__flush-top
+                        {% endif %}">
+            {% if value.categories.page_type == 'activity-log' %}
+                {% import 'activity-log/_activity-list.html' as activity_list with context %}
+                {{ activity_list.render(posts) }}
+            {% elif value.output_5050 %}
+                {% from 'molecules/info-unit-2.html' import info_unit with context %}
+                <div class="o-info-unit-group u-mb30" data-qa-hook="image-text-50-50">
```

Comment:
TODO: This whole block should go into its own template, but to manage the complexity of this PR best break things apart here and do a separate PR to separate this part out.